.prettierrc

```diff
@@ -1,17 +1,17 @@
 {
```

Comment:
the prettierrc wasn't pretty, lol

Reply:
Lol oops, I only ran it recursively within my `aura` and `lwc` folders